rs/xnet/payload_builder/src/lib.rs

```diff
@@ -526,6 +530,7 @@ impl XNetPayloadBuilderImpl {
         }
 
         if !reject_signals.is_empty() {
+            // TODO(MR-635): Change this check to use `MAX_STREAM_MESSAGES` instead.
```

Comment:
I find this TODO confusing since the check does seem to use `MAX_STREAM_MESSAGES` below, but I guess you know what it's about, if you're gonna sort this out soon it's fine by me.

Reply:
It is indeed. Wrote it differently. What I meant to say is the old check uses 200k signals for the check arguing about how many messages could fit in a stream. But since this PR caps them at 10k, that's what we should use for this check.